docsrc/Connecting_and_queries.rst

```diff
@@ -66,13 +67,11 @@ To get started, follow the steps below:
 				engine_name = "your_engine"
 				database_name = "your_database"
 
-				connection = connect(
-					engine_name=engine_name,
-					database=database_name,
-					username=username,
-					password=password,
-				)
-
+				with connect(
+    					engine_name=engine_name,
+    					database=database_name,
+    					auth=UsernamePassword(username, password),
+				) as connection:
 				cursor = connection.cursor()
```

Comment:
Need to indent this line now, since the previous block starts with `with ...`. Otherwise if someone were to copy the code as is, it will be broken right away.
Same below in dotenv section.

Reply:
Added indent